webservices/load_current_murs.py

```diff
@@ -69,13 +108,39 @@ def load_current_murs():
             mur['subject'] = {"text": get_subjects(case_id)}
 
             participants = get_participants(case_id)
-            assign_citations(participants, case_id)
             mur['participants'] = list(participants.values())
-
+            mur['disposition'] = get_disposition(case_id)
             mur['text'], mur['documents'] = get_documents(case_id, bucket, bucket_name)
             # TODO pdf_pages, open_date, close_date, url
+            mur['open_date'], mur['close_date'] = get_open_and_close_dates(case_id)
+
             es.index('docs', 'murs', mur, id=mur['doc_id'])
 
+def get_open_and_close_dates(case_id):
+    with db.engine.connect() as conn:
+        rs = conn.execute(OPEN_DATE, case_id)
+        open_date = rs.fetchone()[0]
+
+        rs = conn.execute(CLOSE_DATE, case_id)
+        close_date = rs.fetchone()[0]
+    return open_date, close_date
+
+def get_disposition(case_id):
+    with db.engine.connect() as conn:
+        rs = conn.execute(DISPOSITION_DATA.format(case_id))
+        disposition_data = []
+        for row in rs:
+            citations = parse_statutory_citations(row[3], case_id, row[2])
```

Comment:
Wouldn't using column names instead of column positions make the code more self-documenting?